stylesheets/components/card.css

```diff
@@ -0,0 +1,122 @@
+.card {
+    font-family: 'Poppins', sans-serif;
+    margin: var(--margin-one);
+    display: flex;
+    max-width: 19rem;
+    height: fit-content;
+    background-color: var(--white);
+    flex-direction: column;
+    border-radius: 0.5rem;
+    box-shadow: 0.25rem 0.25rem 1rem var(--gray-400);
```

Comment:
for values less than 10px, you can use px

Reply:
Updated in the latest commit